test/app.js

```diff
@@ -311,6 +311,22 @@ t.test('App', async t => {
     await ctx.render({json: {valid: result.isValid, errors: result.errors}});
   });
 
+  // PUT /schema/array
+  app.put('/schema/array', async ctx => {
+    const validate = ctx.schema({
+      type: 'object',
+      properties: {
+        test: {type: 'array'}
+      },
+      required: ['test']
+    });
+
+    const data = await ctx.req.json();
+    const result = validate(data);
+
+    await ctx.render({json: {valid: result.isValid, errors: result.errors}});
```

Comment:
This test should also include the changed data, to verify that the result is actually safe after validation.